nrf_wifi/fw_if/umac_if/src/cmd.c

```diff
@@ -208,7 +208,13 @@ enum nrf_wifi_status umac_cmd_init(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 		umac_cmd_data->disable_beamforming = 1;
 	}
 
-	 status = nrf_wifi_hal_ctrl_cmd_send(fmac_dev_ctx->hal_dev_ctx,
+#if defined(CONFIG_NRF_WIFI_QOS_NULL_BASED_RETRIEVAL)
+	umac_cmd_data->ps_data_retrieval_mode = QOS_NULL_FRAME;
+#else
```

Comment:
If you are planning to extend this to also have `Automatic` in the future then shouldn't this be `#else if defined (CONFIG_NRF_WIFI_PS_POLL_BASED_RETRIEVAL)` and then have an else with an error message?